### ISSUE TITLE
[Bug] wxGUI AddWSDialog: "Access window out of range" error message

### ISSUE THREAD

**Issue by nilason** [OSGeo/grass#688]
**Describe the bug**

Adding a WMS layer produces following error message in terminal:

```
ERROR 5: /Volumes/grassdata/srtm/PERMANENT/.tmp/NL.local/17715.2: Access window out of range in RasterIO().  Requested (932,0) of size 466x296 on raster of 933x592.
ERROR 5: /Volumes/grassdata/srtm/PERMANENT/.tmp/NL.local/17715.2: Access window out of range in RasterIO().  Requested (932,296) of size 466x296 on raster of 933x592.
ERROR 5: /Volumes/grassdata/srtm/PERMANENT/.tmp/NL.local/17715.2: Access window out of range in RasterIO().  Requested (1398,0) of size 466x296 on raster of 933x592.
ERROR 5: /Volumes/grassdata/srtm/PERMANENT/.tmp/NL.local/17715.2: Access window out of range in RasterIO().  Requested (1398,296) of size 466x296 on raster of 933x592.
```

but the layer is seemingly rendered correctly.

**To Reproduce**

1. Add web service layer (e.g. from `http://ows.mundialis.de/services/service?`) via Layer Manager
2. See error

**System description (please complete the following information):**

- Operating System: tested on macOS

```
$ g.version -rge
version=7.9.dev
date=2020
revision=230626581
build_date=2020-06-02
build_platform=x86_64-apple-darwin18.7.0
build_off_t_size=8
libgis_revision=f89b009b5
libgis_date=2020-06-02T11:44:09+00:00
proj=6.3.2
gdal=3.1.0
geos=3.8.1
sqlite=3.31.1

Python=3.7.7
wx.version()=4.0.7.post2 osx-cocoa (phoenix) wxWidgets 3.0.5
```

**Additional context**

```
$ g.gisenv set=WX_DEBUG=1
$ g.gui wxpython

# ADD WEB SERVICE LAYER

GUI D1/1: BufferedWindow.UpdateMap(): started (render=False, renderVector=True)
GUI D1/1: RenderMapMgr.Render(): 0 layers to be rendered (force=0, all active layers -> 0)
GUI D1/1: RenderMapMgr.OnRenderDone() time=0.003362 sec (comp: 0.000000)
GUI D1/1: BufferedWindow.UpdateMap(): finished
GUI D1/1: gcmd.RunCommand(): g.region -ugpc
GUI D1/1: gcmd.RunCommand(): get return code 0 (0.040119 sec)
GUI D1/1: BufferedWindow.UpdateMap(): started (render=False, renderVector=True)
GUI D1/1: RenderMapMgr.Render(): 0 layers to be rendered (force=0, all active layers -> 0)
GUI D1/1: RenderMapMgr.OnRenderDone() time=0.000432 sec (comp: 0.000001)
GUI D1/1: BufferedWindow.UpdateMap(): finished
GUI D1/1: gcmd.RunCommand(): g.region -ugpc
GUI D1/1: gcmd.RunCommand(): get return code 0 (0.046875 sec)
GUI D1/1: BufferedWindow.UpdateMap(): started (render=False, renderVector=True)
GUI D1/1: RenderMapMgr.Render(): 0 layers to be rendered (force=0, all active layers -> 0)
GUI D1/1: RenderMapMgr.OnRenderDone() time=0.000260 sec (comp: 0.000001)
GUI D1/1: BufferedWindow.UpdateMap(): finished
GUI D1/1: gcmd.RunCommand(): g.region -ugpc
GUI D1/1: gcmd.RunCommand(): get return code 0 (0.038807 sec)
GUI D1/1: Map.AddLayer(): ltype=wms, command=d.wms wms_version=1.3.0 driver=WMS_GRASS url=http://ows.mundialis.de/services/service? layers=TOPO-WMS styles= format=png srs=4326 maxcols=500 maxrows=500 urlparams= method=nearest map="Topographic WMS - by terrestris"
GUI D1/1: WMTS web service was not found in fetched capabilities file from <http://ows.mundialis.de/services/service?>:
Unable to parse capabilities file. 
                                Tag 'TileMatrixSet' was not found.

GUI D1/1: OnEarth web service was not found in fetched capabilities file from <http://ows.mundialis.de/services/service?>:
Unable to parse tile service file. 
                                 Tag <TiledPatterns> was not found.

GUI D1/1: BufferedWindow.UpdateMap(): started (render=False, renderVector=True)
GUI D1/1: RenderWMSMgr.Render(d.wms wms_version=1.3.0 driver=WMS_GRASS url=http://ows.mundialis.de/services/service? layers=TOPO-WMS styles= format=png srs=4326 maxcols=500 maxrows=500 urlparams= method=nearest map="Topographic WMS - by terrestris"): force=1 img=/var/folders/k4/xy86sh0n0tb3fyjl15kp0ttc0000gn/T/grass7-nila-15970/tmp2s_e_8ol.ppm
GUI D1/1: RenderMapMgr.Render(): 1 layers to be rendered (force=0, all active layers -> 1)
ERROR 5: /Volumes/grassdata/srtm/PERMANENT/.tmp/NL.local/17715.2: Access window out of range in RasterIO().  Requested (932,0) of size 466x296 on raster of 933x592.
ERROR 5: /Volumes/grassdata/srtm/PERMANENT/.tmp/NL.local/17715.2: Access window out of range in RasterIO().  Requested (932,296) of size 466x296 on raster of 933x592.
ERROR 5: /Volumes/grassdata/srtm/PERMANENT/.tmp/NL.local/17715.2: Access window out of range in RasterIO().  Requested (1398,0) of size 466x296 on raster of 933x592.
ERROR 5: /Volumes/grassdata/srtm/PERMANENT/.tmp/NL.local/17715.2: Access window out of range in RasterIO().  Requested (1398,296) of size 466x296 on raster of 933x592.
GUI D1/1: RenderWMSMgr.OnRenderDone(d.wms wms_version=1.3.0 driver=WMS_GRASS url=http://ows.mundialis.de/services/service? layers=TOPO-WMS styles= format=png srs=4326 maxcols=500 maxrows=500 urlparams= method=nearest map="Topographic WMS - by terrestris"): ret=0 time=4.983521
GUI D1/1: gcmd.RunCommand(): g.pnmcomp --o input=/var/folders/k4/xy86sh0n0tb3fyjl15kp0ttc0000gn/T/grass7-nila-15970/tmp2s_e_8ol.ppm mask=/var/folders/k4/xy86sh0n0tb3fyjl15kp0ttc0000gn/T/grass7-nila-15970/tmp2s_e_8ol.pgm opacity=1.0 bgcolor=255:255:255 width=933 height=592 output=/var/folders/k4/xy86sh0n0tb3fyjl15kp0ttc0000gn/T/grass7-nila-15970/tmpul12tc88.ppm
GUI D1/1: gcmd.RunCommand(): get return code 0 (0.086436 sec)
GUI D1/1: RenderMapMgr.OnRenderDone() time=5.073510 sec (comp: 0.086823)
GUI D1/1: BufferedWindow.UpdateMap(): finished
GUI D1/1: gcmd.RunCommand(): g.region -ugpc
GUI D1/1: gcmd.RunCommand(): get return code 0 (0.113265 sec)

$ sips --getProperty pixelHeight --getProperty pixelWidth /var/folders/k4/xy86sh0n0tb3fyjl15kp0ttc0000gn/T/grass7-nila-15970/tmp2s_e_8ol.ppm
/private/var/folders/k4/xy86sh0n0tb3fyjl15kp0ttc0000gn/T/grass7-nila-15970/tmp2s_e_8ol.ppm
  pixelHeight: 592
  pixelWidth: 933

```

The map display size is 933x592: same as downloaded png file.

If I reduce map window size to below 500x500, no error message is issued.

**Comment by tmszi** [comment]
I confirm this bug.

**Comment by petrasovaa** [comment]
Should be fixed with #693.
